storage/cache/subtree_cache.go

```diff
@@ -46,8 +46,10 @@ type SubtreeCache struct {
 	dirtyPrefixes map[string]bool
 	// mutex guards access to the maps above.
 	mutex *sync.RWMutex
-
+	// used to rebuild internal nodes when subtrees are loaded
 	populateSubtree storage.PopulateSubtreeFunc
+	// used for preparation work when subtrees are about to be written to storage
+	prepareSubtreeWrite storage.PrepareSubtreeWriteFunc
```

Comment:
I know it was me who caused this, but it might be worth renaming these two fields to have shorter names?